Code/MinimalLib/jswrapper.cpp

```diff
@@ -522,6 +522,7 @@ EMSCRIPTEN_BINDINGS(RDKit_minimal) {
       .function("draw_to_canvas_with_highlights",
                 &draw_rxn_to_canvas_with_highlights)
 #endif
+      .function("is_valid", &JSReaction::is_valid)
```

Comment:
Why are we adding this function if it's immediately deprecated?

Reply:
True. I had forgotten to expose it earlier, but adding it now is a bit daft.

Reply:
Unexposed it.